tests/test_checkpoint_io/test_hybrid_parallel_plugin_checkpoint_io.py

```diff
@@ -103,6 +99,7 @@ def _preprocess_data(data):
         dist.barrier()
 
     # Check whether the loaded model & optimizer works smoothly.
+    optimizer.zero_grad()
```

Comment:
Why?

Reply:
the sharded model has called backward before,  the grad retained will affect the next check, as the new model has only been trained for one step.